app/controllers/pathways_controller.rb

```diff
@@ -18,21 +18,52 @@ def new
   end
 
   def create
-    @pathway = Pathway.new(pathway_params)
-    @pathway.user = current_user
-    if @pathway.save
+   @pathway = Pathway.new(pathway_params)
+   @pathway.user = current_user
+   if @pathway.save
+      add_tags_on_pathway_create(@pathway)
       redirect_to pathways_path
     else
       render :new, status: :unprocessable_entity
     end
   end
 
+  def add_tags_on_pathway_create(pathway)
+   tags = params[:pathway][:tag_ids]
+   tags.drop(1).each do |tag|
+      Tagging.create!(tag: Tag.find(tag.to_i), pathway: pathway) 
+   end
+  end
+
   def edit
   end
 
   def update
-    @pathway.update(pathway_params)
-    redirect_to pathway_path(@pathway)
+   @pathway.update(pathway_params)
+   edit_pathway_tags(@pathway)
+   redirect_to pathway_path(@pathway)
+  end
+
+  def edit_pathway_tags(pathway)
```

Comment:
these methods should probably be in private (convention is for public methods to be route actions I believe).

Reply:
all done!